src/__mocks__/axios.js

```diff
@@ -0,0 +1,40 @@
+// export default {
+//   get: jest.fn(() => Promise.resolve({ data: 'mocked' })),
+//   post: jest.fn(() => Promise.resolve({ data: 'mocked' })),
+//   create: () => jest.fn(),
+// };
+
+// tslint:disable-next-line:no-empty
+// const mockNoop = () => new Promise(() => {});
+
+// export default {
+//   get: jest.fn(() => Promise.resolve({ data: { total_payout: 100.21 } })),
+//   default: mockNoop,
+//   post: mockNoop,
+//   put: mockNoop,
+//   delete: mockNoop,
+//   patch: mockNoop,
+//   create: () => jest.fn(),
+// };
+
+// export const Mockwakanda = {
+//   post: jest.fn(() => Promise.resolve({ data: 'mocked' })),
+//   get: jest.fn(() => Promise.resolve({ data: 'mocked' })),
+// };
+
+// src/__mocks__/axios.ts
+
+// const mockAxios = jest.genMockFromModule('axios');
+
+// // this is the key to fix the axios.create() undefined error!
+// mockAxios.create = jest.fn(() => mockAxios);
+
+// export default mockAxios;
+// src/__mocks__/axios.ts
+
```

Comment:
@dusmel I think you should delete the commented code above.

Reply:
coool 👍